### ISSUE TITLE
Batch Upscaling Changes File Names

### ISSUE THREAD

**Issue by theraineydaze** [n00mkrad/cupscale#77]
I'm upscaling images that have to keep their exact file name, some of the file names end in numbers like 'r2image.200.png' and 'r2image.300.png' 'r3image.200.png' 

If I upscale these images one at a time the file name stays the same, but if I do multiple images it'll remove the numbers at the end, if certain files become the same name when the numbers are removed one of them is overwritten. So like 'r2image.png' and 'r3image.png'. Turning off overwrite mode doesn't fix this it just adds a suffix to the end while still removing numbers.

**Comment by Tsubajashi** [comment]
Is that issue still appearing if you change ".200" to "-200" for example?

**Comment by theraineydaze** [comment]
Nope! Good idea I could use Bulk Rename Utility for a workaround, thanks!

**Comment by Tsubajashi** [comment]
glad that it works for you :) some software has issues with parsing filenames with dots in it. i have had plenty of issues in my own software and its not even resolved as of yet. (it tries to get the filename, to create a default output folder, and misses "removing" the extension type due to it, and removes everything after the first dot).
Might be related, thats why i share it.

**Comment by cosmicr** [comment]
Just wondering if this bug was ever fixed, or is the project dead?
